analyzer.py

```diff
@@ -77,4 +85,68 @@ def ttest(self):
                         'ts2': entry2['data']
                     })
 
-        return results
\ No newline at end of file
+        return results
+    '''
+    # Later, create a seperate file/function for the server requests.
+    def start_local_server():
+        app = Flask(__name__)
+    '''
+    
+def main():
+    
+    '''
+    Currently not working. Might continue on this afterwards.
+    # Post to Iodide
+    def post_to_iodide():
+
+        url = "http://localhost:8000/customize"      
+        script_path = os.path.dirname(__file__)
+        data_relative_path = "testing/template/template.txt"
+        absolute_file_path = os.path.join(script_path,data_relative_path) 
+
+        files = {'template':open(absolute_file_path,'rb')}
+
+        requests.post(url,files=files)
+        #print(r.text)
+    '''
+
+
+    # Flask localhost with API for Iodide
+    app = Flask(__name__)
+    app.config["DEBUG"]= True
+    
+    @app.route('/data', methods=['GET'])
+    def return_data():
+
+        script_path = os.path.dirname(__file__)
+        data_relative_path = "testing/output/data.json"
+        absolute_file_path = os.path.join(script_path,data_relative_path)
+
+        response = flask.make_response(send_file(absolute_file_path))
+        response.headers['Access-Control-Allow-Origin'] = '*'
+
+        return response
+
+    '''
+    
+    @app.route('/post',methods=['GET'])
+    def send_template():
+        post_to_iodide()
+    
+    '''
+
+    webbrowser.open('upload_file.html')
```

Comment:
When I run the modified changes locally, there are two windows opened by this to the same page doing the same post request. Not sure why it happens though.

Reply:
fixed.
Due to Flask's debug mode, it is detecting changes on 2 processes.